packages/ui/Toggleable/tests/Toggleable-specs.js

```diff
@@ -152,11 +152,14 @@ describe('Toggleable', () => {
 			data.onToggle();
 
 			const expected = 1;
+			const expectedType = {type: 'onToggle'};
```

Comment:
What about adding a case for `toggleProp`? We already have `toggleProp: 'onJiggle'` test case in this file.